### DISCUSSION TITLE
library u8g2 still needed ?

### DISCUSSION THREAD

**Discussion by tk5ep** [dl9rdz/rdz_ttgo_sonde#229] [Q&A]
HI,
I noticed that in the latest versions i've downloaded, the u8g2 libraries are commented in the sketch, so not used.

The Wiki documentation asks to install it. 

BTW, there is a typo on this page : **Arduini** IDE. 

73 TK5EP Patrick

**Comment by dl9rdz** [comment]
Not sure to what your comment "commented in the sketch, so not used" refers to.

The library is necessary.
Its need is also specified in .travis.yml and platformio.ini
It is included by RX_FSK/src/Display.cpp

**Comment by tk5ep** [comment]
OK,

I was talking about the Arduino IDE installation.
In the sketch, the 2 following lines are not used.

`//#include <U8x8lib.h>

//#include <U8g2lib.h>`

And i don't use the /src folder.

The typo is in the Wiki "Arduino IDE installation".

It was just to clarify as I'm writing an article in French on my Web page about your software and IDE installation.

73 Patrick

**Comment by dl9rdz** [comment]
Well, sorry, I am still a bit lost. The sketch (i.e. that should be RX_FSK/RX_FSK.ino) does not contain the two lines you are describing.

Regarding "not using the /src folder", well, I also don't know, there is most of the code, so you basically have to use it. Or the Arduino IDE has to use it. It does so automatically. The sketch contains references to it (e.g. https://github.com/dl9rdz/rdz_ttgo_sonde/blob/07859d7f70d2a93110ccaed4a7b735e4e7a1fa78/RX_FSK/RX_FSK.ino#L19), and also the linker automatically links all code in the src folder to the final image.

Thanks for pointing out the typo, I have fixed it.

**Comment by tk5ep** [comment]
Well, it's my turn to be lost !
I checked what version I was using and it was **devel20210817**
No path ( / src) are added in the includes.

So you meanwhile probably, (certainly) changed something.
I downloaded the latest version and will have a look.

**_updated_** : I loaded the latest version and now, everything makes sense. ;-)

Sorry for the disturbance.

**Comment by dl9rdz** [comment]
Ok, that clarifies a lot :-)

Back then, you are right, there were these comments in the sketch. Also the stuff now in RX_FSK/src was in the libraries folder, and that staff needed the U8g2 library as well.

A lot has changed and improved since then, so using a newer version might be good idea.
